packages/amplication-server/src/core/billing/billing.service.ts

```diff
@@ -110,6 +114,15 @@ export class BillingService {
         successUrl: new URL(cancelUrl, this.clientHost).href,
       },
     });
+    this.analytics.track({
+      userId: workspaceId,
+      event:
```

Comment:
I don't understand this event. You don't only have upgrade or downgrade. You also have a situation where the user didn't do anything

Reply:
I don't understand.... this controller invoke when user click upgrade/downgrade his plan

Reply:
move stigg provider #4856